public/editor/scripts/editor/js/fc/bramble-popupmenu.js

```diff
@@ -22,6 +22,8 @@ define(function(require) {
     self.close = function(e) {
       if(e) {
         e.stopPropagation();
+        self._button$.removeClass('active');
+        $("#editor-pane-nav-options").css("opacity", "0.5");
```

Comment:
Ok. With that addition, I can remove the `$("#editor-pane-nav-options").css("opacity", "0.5");`
and `$("#editor-pane-nav-options").css("opacity", "0.7");` . I still have to use `.addClass('active')` and `.removeClass('active')` with `self._button$`, however. Is that the right way? If not, let me know and i'll figure it out!